xml/Microsoft.VisualC.StlClr.Generic/ReverseBidirectionalIterator`1.xml

```diff
@@ -301,7 +301,7 @@
         <returns>
           <see langword="true" /> if the two iterators are the same object; otherwise, <see langword="false" />.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:Microsoft.VisualC.StlClr.%2A> Gene ric.ReverseBidirectionalIterator`1.Equals(Microsoft.VisualC.StlClr. Gene ric.IInputIterator{`0})?qualifyHint=True&autoUpgrade=False]]></format>
+          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:Microsoft.VisualC.StlClr.%2A> Generic.ReverseBidirectionalIterator`1.Equals(Microsoft.VisualC.StlClr.Generic.IInputIterator{`0})]]></format>
```

Comment:
these links are still broken. the link is being closed in the middle

also this class doesn't seem to have an Equals method

Reply:
In terms of this class not having an Equals method, @mairaw and @JRAlexander, it looks as if there's been a change in the build, so that inherited members have disappeared. Since this is a managed class, it has to have an Equals method. As [MSDN](https://msdn.microsoft.com/en-us/library/bb347663(v=vs.110).aspx) shows, it inherits it from Object. Actually, I don't see any inherited members in the few types I've looked at that have inherited members. There's been a build change that we were unaware of, I think.

Reply:
Oh yeah! You're right @rpetrusha. I hated the previous design but now is much worse. I'll file a LSI. 😢  
So the right link would be to the Object.Equals api

Reply:
@mairaw, yes. I just sent an email, but this appears to be true of all inherited members of all types -- they've disappeared from the docs.